src/renderers/WebGLRenderer.js

```diff
@@ -407,7 +407,7 @@ function WebGLRenderer( parameters = {} ) {
 		_canvas.width = Math.floor( width * _pixelRatio );
 		_canvas.height = Math.floor( height * _pixelRatio );
 
-		if ( updateStyle !== false ) {
+		if ( updateStyle ) {
```

Comment:
While you are already at it:
```js
if ( updateStyle === true ) {
```
We tend to prefer the more explicit style in `if` statements.